docs/_blog/2025_q2_update.md

```diff
@@ -0,0 +1,55 @@
+
+Here are some of the notable releases and changes over Q2 to the software libraries we maintain:
+- **The bdk_wallet 2.0 release!** Following on the heels of 1.0, we've released [BDK 2.0](https://github.com/bitcoindevkit/bdk_wallet/releases/tag/wallet-2.0.0) which includes a bug fix for handling stuck or evicted transactions, performance enhancements for large wallets, more extensive test coverage, and the return of TxDetails.
+- **BDK CLI 1.0.** Our [command-line interface has reached its 1.0 milestone](https://github.com/bitcoindevkit/bdk-cli/releases/tag/v1.0.0), uses bdk_wallet 1.0.0 and integrates Kyoto, which uses the Kyoto protocol for compact block filters. It sets SQLite as the default database and drops support for sled.
+- **Language Bindings version 1.2 for iOS, Android, and Python 1.2** The [bdk-ffi 1.2 release](https://github.com/bitcoindevkit/bdk-ffi/releases/tag/v1.2.0) brings Compact Block Filter support through our Kyoto integration, making privacy-preserving light clients accessible to mobile developers. We've also published comprehensive [API documentation](https://bitcoindevkit.org/docs/) for Java and Android bindings.
+
+There are also new projects and initiatives being built.
+- **Silent Payments.** The new [bdk-sp](https://github.com/bitcoindevkit/bdk-sp) crate brings BIP352 Silent Payments functionality to BDK. It has delivered a complete CLI implementation with wallet initialization, PSBT creation and signing, and blockchain scanning capabilities, and full compatibility with BIP352 test vectors and integration examples with bdk-tx.
+- **Transaction Building.** The experimental [bdk-tx](https://github.com/bitcoindevkit/bdk-tx) project represents an evolution of transaction building. This effort to decouple transaction building from the wallet, using rust-miniscript's planning module for optimal satisfaction-weight calculations and the new bdk_coin_select crate for policy-aware coin selection.
+- **Streaming Electrum Client.** The [electrum_streaming_client](https://github.com/bitcoindevkit/electrum_streaming_client) is a streaming, sans-IO Electrum client for asynchronous and blocking Rust applications. This crate provides low-level primitives and high-level clients for communicating with Electrum servers over JSON-RPC. It supports both asynchronous (futures/tokio) and blocking transport models.
+
+## Our Grantees in Action
+
+We're excited to welcome new Silver [corporate members](https://bitcoindevkit.org/blog/_2025q1-new-foundation-members/) the BDK Foundation and thank them for their financial support!
+- AnchorWatch
+- CleanSpark
+- Proton Foundation
+
+We're also excited to welcome new Associate members who are supporting the BDK Foundation by training & funding developers working on BDK!
+- Btrust
+- Vinteum
```

Comment:
I think some links to these companies/orgs would be cool and give them a bit of visibility (even though most people probably know who they are).

Reply:
great great idea, updated with a link for each